misc/docker-integration/integration_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build docker
+// +build docker
```

Comment:
Since we have set go version to 1.19 in go.mod, I don't think we need to keep backward compatibility for the build constraint directives. Having only `//go:build` should be enough.